### PR TITLE
Electron version upgrade to 28.0.0

### LINKED ISSUE #1001: Crashes (segfaults) on Wayland
**Describe the bug**
Ever since the last update (from v1.3.12 to v1.3.14), I am fairly frequently getting crashes when running on Wayland (often when I am in a call). Although the crashes can occur randomly, I can reproduce it fairly consistently using the steps below.

**To Reproduce**
1. Launch in a terminal with the following command
   ```
   teams-for-linux --enable-features=UseOzonePlatform --ozone-platform=wayland
   ```
2. Quickly maximise the application before it has time to load fully.
3. It will crash and and I see this output in my terminal
   ```
   [ERROR] net::ERR_CONNECTION_REFUSED
   [1]    52878 segmentation fault (core dumped)  teams-for-linux --enable-features=UseOzonePlatform --ozone-platform=wayland 
   ```

**Expected behavior**
No crash or segfault like in v1.3.12.

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: Fedora 38 
 - Installation package: RPM
 - Version: 1.3.14
 - Desktop Environment: KDE Plasma 5.27.8
 - Kernel Version: 6.5.8-200.fc38.x86_64 (64-bit)
 - Graphics Platform: Wayland

**Additional context**
I've been experiencing a similar issue with VSCode which is also electron based under certain conditions (microsoft/vscode#184124) which has been blamed on this upstream electron issue (electron/electron#37531) and I think this might be the same issue. For now, I will stick with v1.3.12.

### PR DESCRIPTION
**Pull request by MAFLO321**
Update dependencies to latest versions

Fixes #1001

### CODE REVIEW

**Comment by MAFLO321** [comment]
@IsmaelMartinez Maybe first create a pre-release, please. I have just tested it a bit on my system.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "1.3.26",
+  "version": "1.3.27",
```
Comment (IsmaelMartinez) [inline package.json]:
```suggestion
  "version": "1.4.0",
```

I think is worth starting upping the second number when updating electron to a mayor version, as we had a few problems and remembering 1.3.xx is a bit of a pain.

File: com.github.IsmaelMartinez.teams_for_linux.appdata.xml [hunk 2]
```diff
@@ -14,6 +14,14 @@
 	<url type="bugtracker">https://github.com/IsmaelMartinez/teams-for-linux/issues</url>
 	<launchable type="desktop-id">com.github.IsmaelMartinez.teams_for_linux.desktop</launchable>
 	<releases>
+		<release version="1.3.27" date="2023-12-21">
```
Comment (IsmaelMartinez) [inline com.github.IsmaelMartinez.teams_for_linux.appdata.xml]:
```suggestion
		<release version="1.4.0" date="2023-12-21">
```

as mention in the other message

**Comment by IsmaelMartinez** [comment]
building just now. I will mark it as pre-release as soon as it is done generating the artefacts. Thanks a lot for your contribution!

**Pull request merged**